tests/python/unittest/test_meta_schedule_runner.py

```diff
@@ -690,6 +690,8 @@ def _check_correct_add(args_before: List[np.ndarray], args_after: List[np.ndarra
         a_before, b_before, c_before = args_before
         a_after, b_after, c_after = args_after
         c_before = a_before + b_before
+        print(a_before)
```

Comment:
remove the print

Reply:
there are two more prints below this in the same file that likely need to be removed as well.